llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -2022,6 +2022,8 @@ let Predicates = [HasPAuthLR] in {
     //                              opcode2, opcode,   asm
     def AUTIASPPCr : SignAuthOneReg<0b00001, 0b100100, "autiasppcr">;
     def AUTIBSPPCr : SignAuthOneReg<0b00001, 0b100101, "autibsppcr">;
+  }
+  let Defs = [X17], Uses = [X15, X16, X17] in {
```

Comment:
This bug fix in the description of which registers instructions {PAC|AUT}I{A|B}171615 read and write should be upstreamed in a separate PR.

Reply:
The separate PR for this fix is now created: #123354

Reply:
... and merged.